packages/e2e-tests/specs/a11y.test.js

```diff
@@ -29,6 +30,36 @@ describe( 'a11y', () => {
 		expect( isFocusedToggle ).toBe( true );
 	} );
 
+	it( 'checks persistent selection', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'Testing editor selection persistence' );
+
+		await page.keyboard.down( 'Control' );
```

Comment:
There is also this handy helper method which can simplify this code to:
```js
await pressKeyWithModifier( 'ctrlShift', '`' );
await pressKeyWithModifier( 'ctrlShift', '`' );
```

It would be nice to add a code comment for those who don't know this keyboard shortcut.

Just noting that it moves focus backwards.